pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -123,6 +149,13 @@ func StatefulSetForCluster(
 									ContainerPort: util.DefaultCqlPort,
 								},
 							},
+							VolumeMounts: []apiv1.VolumeMount{
+								{
+									Name:      sharedVolumeName,
+									MountPath: sharedVolumeMountPath,
+									ReadOnly:  false,
```

Comment:
(nit) can we make this ReadOnly: true? Although ES also has readOnly: false set, so can be a follow up/pending discussion

Reply:
Good idea. Done. For ElasticSearch too.